.github/workflows/release.yml

```diff
@@ -23,6 +23,13 @@ jobs:
           registry-url: 'https://registry.npmjs.org'
         if: ${{ steps.release.outputs.release_created }}
       - run: yarn install --frozen-lockfile
+      - name: Generating Lib types
+        run: yarn generate
+      - name: Commit yarn generated files
+        uses: stefanzweifel/git-auto-commit-action@v2.3.0
+        with:
+          commit_message: Commit yarn generated files
```

Comment:
Please change this commit message to something like this "chore: generate library types" to be compatible with the [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/) spec